src/scripts/ui/navigation.js

```diff
@@ -17,8 +17,12 @@ document.addEventListener('DOMContentLoaded', () => {
   document.body.addEventListener('click', () => {
```

Comment:
@wesrice or @kevcenteno I'm not sure if this is the best way to mimic how the search bar works on the dev site with hiding the rest of the header contents when the search bar is focused, but disappearing when it is no longer focused.

Reply:
Very, neat thanks! I removed the visibility from the `navigation.js` file and added the following to `.header.scss` 

```
.header:focus-within .header-search-mobile,
.header:focus-within .header-module-toggle {
  display: none;
}
```

But is there a CSS only way to add and remove the `search-mobile-open`? Or can this only be done with javascript?

Reply:
![Screen Shot 2021-10-11 at 9 29 18 AM](https://user-images.githubusercontent.com/52248962/136798404-eb364d36-64c3-4493-8606-c627dfee50b0.png)

Actually, one issue I noticed with this method is in the screenshot. Even though the element isn't focused anymore the search bar and the two buttons are still being shown. It isn't until I click anywhere on the page itself does it revert back to how it should.